bot/services/class_service.py

```diff
@@ -0,0 +1,392 @@
+import logging
+
+import discord
+from discord import CategoryChannel
+
+import bot.bot_secrets as bot_secrets
+from bot.consts import Colors
+from bot.data.class_repository import ClassRepository
+from bot.messaging.events import Events
+from bot.models.class_models import ClassSemester, ClassChannel, ClassChannelScaffold
+from bot.services.base_service import BaseService
+from bot.sock_bot import SockBot
+from bot.utils.helpers import error_embed, fetch_optional_message
+
+log = logging.getLogger(__name__)
+MAX_CHANNELS_PER_CATEGORY = 50
+WELCOME_MESSAGE_REACTION = '✅'
```

Comment:
IMO I would put log below these constants and separate them with an extra newline but that's IMO